_studio/mfx_lib/scheduler/linux/src/mfx_scheduler_core_task_management.cpp

```diff
@@ -593,7 +593,8 @@ void mfxSchedulerCore::MarkTaskCompleted(const MFX_CALL_INFO *pCallInfo,
     // wake up external threads waiting for a free task object
     if (taskReleased)
     {
-        m_freeTasks.Signal(1);
+        ++m_freeTasksCount;
```

Comment:
can you, please, double check that this is being executed under mutex?

Reply:
Fixed